native/src/components/ThemeContext.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { createContext, ReactElement, useMemo, useState } from 'react'
+import { ThemeProvider } from 'styled-components/native'
+
+import { ThemeType } from 'build-configs/ThemeType'
+
+import buildConfig from '../constants/buildConfig'
+
+type ThemeKey = 'light' | 'contrast'
```

Comment:
🙃 I guess you could make this a shared var in build configs somewhere to use it both for web and native.